views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -756,14 +786,18 @@ define([
              */
             updateResetCount: function updateResetCount() {
                 var remaining = this.config.maxRecords - this._recordsAttempts - 1,
-                    resetLabel = resetIcon;
+                    resetLabel = resetIcon,
+                    canRecordAgain;
 
                 if (this.config.maxRecords > 1) {
                     resetLabel += ' (' + remaining + ')';
                 }
                 if (this.controls.reset) {
                     this.controls.reset.updateLabel(resetLabel);
                 }
+                // reflect can-record-again state in the DOM
+                canRecordAgain = this.config.maxRecords === 0 || remaining >= 0;
```

Comment:
It will still record again after you open-close overview, even though MediaInteraction's maxPlays are respected. (remaining-records aren't even saved in itemState)
I guess it's not in the scope of this PR.